charts/seed-monitoring/charts/core/charts/prometheus/rules-tests/kube-etcd3.rules.test.yaml

```diff
@@ -27,10 +27,10 @@ tests:
   - series: 'etcdbr_snapshot_duration_seconds_count{job="kube-etcd3-backup-restore",role="main",kind="Incr",succeeded="true"}'
     values: '0+0x30'
   # KubeEtcdFullBackupFailed
+  - series: 'etcdbr_snapshot_latest_timestamp{job="kube-etcd3-backup-restore",role="main",kind="Full"}'
+    values: '1000000000+0x30'
   - series: 'etcdbr_snapshot_latest_revision{job="kube-etcd3-backup-restore",role="main",kind="Full"}'
-    values: '0+1x3000'
-  - series: 'etcdbr_snapshot_duration_seconds_count{job="kube-etcd3-backup-restore",role="main",kind="Full",succeeded="true"}'
-    values: '0+0x3000'
+    values: '1000084601+0x30'
```

Comment:
Shouldn't we set "etcdbr_snapshot_required" here?  Probably that's why test is failing.

Reply:
correct, I will update the tests